celery/app/base.py

```diff
@@ -1,5 +1,6 @@
 """Actual App instance implementation."""
 import inspect
+import multiprocessing
```

Comment:
as celery use billiard which is a fork of multiprocessing,, should we use multiprocessing here? can  multiprocessing.current_process() imported from billiard?

Reply:
Did not realize it was a thing, will switch to billiard

Meanwhile fixed flake8 (cannot open another fork atm, committed here) https://github.com/pycqa/flake8/issues/1689